src/main/resources/application.conf

```diff
@@ -337,6 +337,10 @@ mantis {
     # Doc: https://github.com/ethereum/EIPs/blob/master/EIPS/eip-161.md
     eip161-block-number = "1000000000000000000"
 
+    # EIP-609 fork block number (ETH Only)
+    # Doc: https://github.com/ethereum/EIPs/blob/master/EIPS/eip-609.md
+    byzantium-block-number = "1000000000000000000"
```

Comment:
It is already in config ( https://github.com/input-output-hk/mantis/pull/491/files#diff-5eab1f3fccda52c11d4e4dbdd0959523R363)

Reply:
good catch, missed in merge